modules/js/perf/perf_imgproc/perf_sobel.js

```diff
@@ -88,44 +96,48 @@ function perf() {
         let suite = new Benchmark.Suite;
         totalCaseNum = 0;
         currentCaseId = 0;
+        let params = "";
+        let paramObjs = [];
+        paramObjs.push({name:"size", value:"", reg:[""], index:0});
+        paramObjs.push({name:"ddepth", value:"", reg:["/CV\_[0-9]+[A-z]+[A-z]+[0-9]/g"], index:1});
+        paramObjs.push({name:"dxdy", value:"", reg:["/\\([0-2]+,[0-2]\\)/"], index:2});
```

Comment:
First part of the parameter is also single digit

Reply:
I have simplified it. Please review it.